### ISSUE TITLE
missing core os option

### ISSUE THREAD

**Issue by vibrantleaf** [FedoraQt/MediaWriter#682]
i was about to install a fedora core os , when i went to select the release in media writer i found out that it was missing as an option , is this intentional or a bug , or could fedora core os be added as an option?

the media writer version i was using was: 5.0.9 on windows

**Comment by grulja** [comment]
It looks CoreOS is not available in the[ json file](https://fedoraproject.org/releases.json) we use to download images from. This makes it harder to support.

**Comment by gridhead** [comment]
I have been in conversation with @hrismarin and @cverna throughout today on this. 

We can achieve this using the readily available [stream metadata](https://docs.fedoraproject.org/en-US/fedora-coreos/stream-metadata/). But being able to do something does not always mean that we should do something. The project would require major rewiring to be able to understand the [ignition configuration](https://github.com/coreos/ignition) and establish them on Fedora CoreOS native installations. [CoreOS Installer](https://github.com/coreos/coreos-installer) exists that does that much better than what we would be able to do here. We can restrict our implementation to burning unconfigured untouched [Fedora CoreOS ISO images](https://fedoraproject.org/coreos/download/) to the flash drive, but given that the variant is intended for advanced users, I am not sure just how worth it would be.

@grulja, what do you think?

**Comment by grulja** [comment]
I agree. FMW is primarily designed for regular users. Adding CoreOS specific features would introduce unnecessary complexity that is not really useful for our target audience.

**Comment by ApolloDawnstar** [comment]
Just to bluntly speak for everyone, CoreOS is an absolute pain in the ass to install, even for technical users. This feature is repeatedly asked for because people want a simpler way to put a bog standard CoreOS install on a USB stick. The current procedure is, in my opinion, unnecessarily complex, and it's easy to end up with an unbootable USB even if you follow the instructions.

Even if Media Writer isn't the home for that kind of solution, fine, but I think Fedora as an organization should find some better way to get people moving with a bootable CoreOS USB. CoreOS is fantastic for its auto update system and immutability, but I shouldn't have to manually make config files and run specific commands just to get something basic working. Just ask me for a username and a public key and send me on my way, y'know?

**Comment by beedell-roke** [comment]
@grulja, we've already obscured context menus for non-default OS options, where we can add COS, so I advise leaving this open until [`coreos/fedora-coreos-tracker/issues/566`](https://github.com/coreos/fedora-coreos-tracker/issues/566#issuecomment-5254988454) has been resolved.
